test/testdata/rbi/enumerable.rb

```diff
@@ -19,3 +19,15 @@
 [1, 3, 20].sort_by {|n| [n.to_s, [1, 2]]}
 
 T.assert_type!([1].lazy, Enumerator::Lazy[Integer])
+
+# There are 3 different ways to call all?, any? and none?
+a = [1, 3, 20]
```

Comment:
The pattern as an argument is the short version of doing `... { |e| e === f }`, so under the hood `===` is invoked without having to open up a block. You could adapt a bit the tests to reflect that operation.

Reply:
> `one?` also supports passing a pattern as argument eyes

Good catch, thanks @sebastian-palma. I don't think I'll have time to to further work on this PR today, will look into it next week.

Reply:
@sebastian-palma, I've added `one?` now. I'm actually not sure what you mean about adapting the tests...

Reply:
Sorry if I wasn't clear @orangethunder. What I meant is you're just passing every element in each `all?/any?/none?/one?` block and that's only checking the truthiness of every object, which is something you can do without opening the block; from the docs "_If the block is not given, Ruby adds an implicit block of { |obj| obj } which will cause all? to return true when none of the collection members are false or nil._".

So, in the examples what you're doing with `all?(1)` is to tell Ruby to do `1 === 1`, which given the elements in `a` only `any?` and `one?` correspond themselves with their `method?(pattern)` version.

What I suggest is adapting them to return the same given the block and the pattern (without block) version.

Reply:
@sebastian-palma, I think the purpose of this test is only to see that the sigs are correct?